.github/workflows/chewbbaca.yml

```diff
@@ -2,17 +2,17 @@ name: chewbbaca
 
 on:
   push:
-    branches: [ main ]
+    branches: [ dev_SchemaEvaluator ]
```

Comment:
I think this will remove the main branch from automated tests wit GitHub Actions.

Reply:
Indeed it will! I think this was a leftover from testing.. :P